# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../howl/howl-0.9.8-darwin-locking-api.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Make the exposed API for Darwin similar to that on the other platforms, so
applications using it will be portable.

  - Hans Petter Jansson <hpj@novell.com>

diff -upr howl-0.9.8-orig/src/lib/howl/MacOSX/macosx_salt.c howl-0.9.8/src/lib/howl/MacOSX/macosx_salt.c
--- howl-0.9.8-orig/src/lib/howl/MacOSX/macosx_salt.c	Fri Jan 28 11:27:36 2005
+++ howl-0.9.8/src/lib/howl/MacOSX/macosx_salt.c	Fri Jan 28 11:32:09 2005
@@ -229,3 +229,19 @@ sw_socket_error_code()
 {
 	return errno;
 }
+
+
+sw_result
+sw_salt_lock(
+				sw_salt	self)
+{
+	sw_assert(self);
+}
+
+
+sw_result
+sw_salt_unlock(
+				sw_salt	self)
+{
+	sw_assert(self);
+}
